Makefile

```diff
@@ -1,3 +1,79 @@
+# Default shell
+SHELL := /bin/sh
+
+# Global CI
+LANGUAGES := go
+# Global CI. Do not edit
+MAKEFILE := Makefile.docs
+CI_REPO_URL := https://github.com/src-d/ci.git
+SHARED_PATH ?= /etc/shared
+CI_PATH ?= $(SHARED_PATH)/ci
+
+$(MAKEFILE):
+	@if [ ! -r "./$(MAKEFILE)" ]; then \
+		if [ ! -r "$(CI_PATH)/$(MAKEFILE)" ]; then \
+			echo "Downloading 'ci'..."; \
+			rm -rf "$(CI_PATH)"; \
+			mkdir -p "$(CI_PATH)"; \
+			git clone "$(CI_REPO_URL)" "$(CI_PATH)"; \
+		fi; \
+		echo "Installing 'ci'..."; \
+		cp $(CI_PATH)/$(MAKEFILE) .; \
+	fi; \
+
+-include $(MAKEFILE)
+
+# Conf vars
+GITHUB_API_KEY ?=
+LOGS_PATH ?= /var/log/docsrv
+LANDING_PATH ?= $(SHARED_PATH)/landing
+INIT_PATH ?= $(shell pwd)/../ci/docs/init.d
+CONF_PATH ?= $(shell pwd)/../ci/docs/conf.d
+
+#vars
+docker_container_name = docsrv-instance
+docker_image_name = docsrv-image
+
+#tools
+mkdir := mkdir -p
+goBuild := CGO_ENABLED=0 go build -o
```

Comment:
@Serabe 
regarding to upper vs lower, you can read my [comment#r120616180](https://github.com/src-d/docsrv/pull/13#discussion_r120616180)